.goreleaser.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 the Heptio Ark contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dist: _output
+builds:
+  - main: ./cmd/ark/main.go
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      # don't build arm/arm64 for darwin or windows
+      - goos: darwin
+        goarch: arm
+      - goos: darwin
+        goarch: arm64
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
+    ldflags:
+      - -X "github.com/heptio/ark/pkg/buildinfo.Version={{ .Version }}" -X "github.com/heptio/ark/pkg/buildinfo.GitSHA={{ .Env.GIT_SHA }}" -X "github.com/heptio/ark/pkg/buildinfo.GitTreeState={{ .Env.GIT_TREE_STATE }}"
```

Comment:
Instead of `ENV.GIT_SHA`, could you use `{{ .FullCommit }}`?

Should we consider removing the git tree state?

Reply:
I'd be fine with both of these changes.  Currently, GitSHA shows the tag name for tagged releases, which is redundant with the version. I'd rather show an actual SHA (which .FullCommit will do).  Also, goreleaser will error out if the tree state is dirty. So probably fine to remove that too.

Reply:
@ncdc I think i'll leave this as-is for now and tackle the change after the beta ships

Reply:
👍